pages/profile/read/views/index.jsx

```diff
@@ -41,6 +41,8 @@ const getPremises = roles => {
 
 const Index = ({
   isUser,
+  isAdmin,
+  profileRole,
```

Comment:
This shouldn't be checking `isAdmin` but that the user has permissions to perform the `profile.permissions` action.

`user.allowedActions.includes('profile.permissions')` (or similar) should do it.

Take a look at the establishment dashboard for an example.

Reply:
Here's an example of using `allowedActions` https://github.com/UKHomeOffice/asl-pages/blob/master/pages/place/list/views/index.jsx#L54-L72